source/gameengine/Converter/BL_BlenderDataConversion.cpp

```diff
@@ -904,7 +900,7 @@ static void BL_CreatePhysicsObjectNew(KX_GameObject* gameobj,
 	}
 }
 
-static KX_LodManager *lodmanager_from_blenderobject(Object *ob, KX_Scene *scene, KX_BlenderSceneConverter *converter, bool libloading)
+static KX_LodManager *lodmanager_from_blenderobject(Object *ob, KX_Scene *scene, KX_BlenderSceneConverter& converter, bool libloading)
```

Comment:
I set the KX_BlenderSceneConverter to reference instead of pointer to forbid store it in a class without copying it.